### PR TITLE
Docs improvements (Consume and Reveal blinded UTXO) 

### LINKED ISSUE #13: Example for transfer accept
Currently in the demo script, the BLINDING factor on the other machine isn't used, nor is any form of accept transfer command is called (on the other machine). It would help to have an example of how this is done.

### PR DESCRIPTION
**Pull request by crisdut**
Closes #13

### CODE REVIEW

**Comment by dr-orlovsky** [comment]
Asking @zoedberg to review & confirm that this is the way the system is intended to work

File: contrib/demo-rgb.sh [hunk 1]
```diff
@@ -158,3 +158,34 @@ btc-cold finalize --publish testnet ${PSBT}
...
+# The close method used in PAYMENT operation.
+CLOSE_METHOD="tapret1st"
+
+# Next, we need to compose the reveal information to unlock the uxto.
+REVEAL="$CLOSE_METHOD@$RECEIVE_UXTO#$BLINDING_FACTOR"
+
+# Let's consume and reveal the concealed seal inside the consignment file.
+rgb -n testnet transfer consume ${CONSIGNMENT} --reveal ${REVEAL}
+
+# Now, we need check if contract state changed.
```
Comment (zoedberg) [inline contrib/demo-rgb.sh:181]:
```suggestion
# Now, we need to check if contract state has changed.
```

File: contrib/demo-rgb.sh [hunk 2]
```diff
@@ -158,3 +158,34 @@ btc-cold finalize --publish testnet ${PSBT}
 
 # Now, once the transaction will be mined, the verification should pass
 rgb consignment validate ${CONSIGNMENT}
+
+# -- CONSUME AND UNLOCK ASSET ------------------------------------------------
+# Go to a remote server / other machine and do the following
+
+# First, we must get all the information required to consume the consignment file.
```
Comment (zoedberg) [inline contrib/demo-rgb.sh:165]: could you please remove spaces at end of lines? I've noticed it in this and other 4 lines (I suggest highlighting trailing whitespace in your editor or setting `git config --global diff.wsErrorHighlight all`)

**Review by zoedberg — CHANGES_REQUESTED** [review]
Hi @crisdut,

thanks, overall looks good, I left just a few notes.

File: contrib/demo-rgb.sh (hunk 2 above)
Comment (crisdut) [reply contrib/demo-rgb.sh]: WOW, thanks. This trick is usefully

File: contrib/demo-rgb.sh [hunk 3]
```diff
@@ -158,3 +158,34 @@ btc-cold finalize --publish testnet ${PSBT}
...
+# The blinding factor generated in blinded utxo operation.
+BLINDING_FACTOR=$INVOICE_BLINDING
+# The close method used in PAYMENT operation.
+CLOSE_METHOD="tapret1st"
+
+# Next, we need to compose the reveal information to unlock the uxto.
+REVEAL="$CLOSE_METHOD@$RECEIVE_UXTO#$BLINDING_FACTOR"
+
+# Let's consume and reveal the concealed seal inside the consignment file.
+rgb -n testnet transfer consume ${CONSIGNMENT} --reveal ${REVEAL}
```
Comment (zzdhidden) [inline contrib/demo-rgb.sh:179]: The command  is `rgb-cli`

Comment (crisdut) [reply contrib/demo-rgb.sh:179]: Yes, you are correct!

I will change it!

Thanks!

**Pull request merged**